src/commands/plzCommand.ts

```diff
@@ -0,0 +1,29 @@
+import * as plz from '../please';
+
+import { argumentPrompt } from './utils';
+
+export async function plzCommand(args: {
+  command: string;
+  args?: string[];
+  runtime?: boolean;
+}): Promise<void> {
+  const { command, args: commandArgs = [], runtime = false } = args;
+
+  let runtimeArgs: string | undefined;
+  if (runtime) {
```

Comment:
Might make more sense to only prompt for `plz run`. For `plz test`, personally I would expect it to just run all the tests.

Reply:
As discussed IRL, I've removed the prompt for `plz test`.